### PR TITLE
refactor(rxTags): move Components/rxTags to Elements/Tags

### PR DESCRIPTION
**Pull request by kevincareer427**
JIRA: https://jira.rax.io/browse/FRMW-625

### LGTMs
- [x] Dev LGTM
- [x] Dev LGTM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +0,0 @@
...
-
- - [Contribution Guide](./CONTRIBUTING.md)
- - [Versioning Guide](./guides/versioning.md)
- - [Changelog](./CHANGELOG.md)
- - [Developer Setup](./guides/ui-setup.md)
- - [Introduction to Testing](./guides/testing.md)
- - [Common Questions](./guides/common-questions.md)
- - [CSS Style Guide](./guides/css-styleguide.md)
- - [JS Styleguide](./guides/js-styleguide.md)
- - [Task Board](https://waffle.io/rackerlabs/encore-ui)
```
Comment (Droogans) [inline README.md]: This old thing was overrated anyhow!

`</sarcasm>`

Hey, just kidding. I think you accidentally deleted this.

Comment (kevincareer427) [inline README.md]: @Droogans haha, found that too

File: src/elements/Tags/rxTags.meta.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+    "displayName": "rxTags",
+    "stability": "stable",
+    "description": "apply predetermined descriptions to an entity",
+    "api": "directive:rxTags",
+    "keywords": [
+      "rxTags",
+      "tags",
+      "tag",
+      "cloud"
```
Comment (100stacks) [inline src/elements/Tags/rxTags.meta.json]: Please sort keywords alphabetically.

File: src/elements/Tags/docs/examples/tags.simple.html [hunk 3]
```diff
@@ -1,5 +1,4 @@
-<!-- Sample HTML goes here as a live example of how the component can be used -->
-<div ng-controller="rxTagsCtrl">
+<div ng-controller="tagsSimpleExampleCtrl">
     <h4>Standard:</h4>
     <rx-tags
```
Comment (100stacks) [inline src/elements/Tags/docs/examples/tags.simple.html]: @kevincareer427, please update to `2 spaces` indentation: 

http://rackerlabs.github.io/encore-ui/#/guides/html-styleguide

**Comment by CITguy** [comment]
Please remove `rxTags.module.js`

File: src/elements/Tags/docs/examples/tags.simple.html (hunk 3 above)
Comment (kevincareer427) [inline src/elements/Tags/docs/examples/tags.simple.html]: @100stacks updated

**Comment by parlarjb** [comment]
LGTM

**Comment by 100stacks** [comment]
LGTM

**Pull request merged**